spring-cloud-skipper-client/src/main/java/org/springframework/cloud/skipper/client/DefaultSkipperClient.java

```diff
@@ -238,6 +240,43 @@ public Release scale(String releaseName, ScaleRequest scaleRequest) {
 		return resourceResponseEntity.getBody();
 	}
 
+	@Override
+	public String getFromActuator(String releaseName, String appName, String appId, String endpoint) {
+
+		Map<String, Object> uriVariables = new HashMap<>();
+		uriVariables.put("releaseName", releaseName);
+		uriVariables.put("appName", appName);
+		uriVariables.put("appId", appId);
+
+		ResponseEntity<String> resourceResponseEntity =
+				restTemplate.exchange(
+						baseUri + "/release/actuator/{releaseName}/{appName}/{appId}?endpoint=" + endpoint,
+						HttpMethod.GET,
+						null,
+						String.class,
+						uriVariables);
+
+		return resourceResponseEntity.getBody();
+	}
+
+	@Override
+	public Object postToActuator(String releaseName, String appName, String appId, ActuatorPostRequest request) {
+		Map<String, Object> uriVariables = new HashMap<>();
+		uriVariables.put("releaseName", releaseName);
+		uriVariables.put("appName", appName);
+		uriVariables.put("appId", appId);
+
+		HttpEntity<ActuatorPostRequest> httpEntity = new HttpEntity<>(request);
+		ResponseEntity<String> resourceResponseEntity =
+				restTemplate.exchange(baseUri + "/release/actuator/{releaseName}/{appName}/{appId}",
+						HttpMethod.POST,
+						httpEntity,
+						String.class,
+						uriVariables);
+
+		return resourceResponseEntity.getBody();
```

Comment:
I believe this is always returning a `String` but the API return is typed to `Object`. The `get` counterpart also returns `String`.

Reply:
Get returns a json string, otherwise, the user needs to pass the response type which would be a PITA.  Similar reasoning for post, but user isn't always concerned about the response body, just the http status.

Reply:
Yep, I was pretty sure that the main reasoning for dropping the Type support was "PITA". :) 

Might we worth calling out something about these return types in the javadocs (if you did not already).